server/cp/output/formatter/newsmlg2.py

```diff
@@ -0,0 +1,29 @@
+
+import cp
+
+from superdesk.publish.formatters.newsml_g2_formatter import NewsMLG2Formatter, SubElement
+
+
+class CPNewsMLG2Formatter(NewsMLG2Formatter):
+
+    def _format_headline(self, article, content_meta):
+        """Appends the headline element to the contentMeta element
+
+        :param dict article:
+        :param Element content_meta:
+        """
+        try:
+            headline = article['extra'][cp.HEADLINE2]
+            SubElement(content_meta, 'headline').text = headline
+            SubElement(content_meta, 'headline', attrib={'role': 'short'}).text = article.get('headline', '')
```

Comment:
If there is no `article['extra'][cp.HEADLINE2]` we also don't fill a short headline, right?

Reply:
yep if there is just short there will be only 1 in the output